crates/re_types/definitions/rerun/archetypes/view_coordinates.fbs

```diff
@@ -7,6 +7,8 @@ namespace rerun.archetypes;
 
 
 /// How we interpret the coordinate system of an entity/space.
+///
+/// \example view_coordinates_simple
```

Comment:
We should leave a not what we do if no ViewCoordinates is present, i.e. how we interpret the coordinate system of a space in absence of explicit specification